packages/replayio/src/commands/record.ts

```diff
@@ -19,7 +19,11 @@ registerCommand("record", { checkForRuntimeUpdate: true, requireAuthentication:
 async function record(url: string = "about:blank") {
   const prevRecordings = await getRecordings();
 
-  await launchBrowser(url, { processGroupId: uuid() });
+  const didLaunchBrowser = await launchBrowser(url, { processGroupId: uuid() });
+  if (!didLaunchBrowser) {
+    // Browser is missing or somehow corrupted
+    await exitProcess(1);
```

Comment:
This should be accompanied by some log - even if it's just "Something whent terribly wrong". Right now the process exits and the terminal stays blank - leaving me confused as to what has happened.

Reply:
I addressed this as part of https://github.com/replayio/replay-cli/pull/386 (that temporarily targets your PR)

Reply:
I'm guessing you just made the launch command throw to test this?

I tested it, and noticed that we would have already printed an error when the upgrade failed (which we would have required, if the install was missing). But I guess you're right that maybe the file could be there but fail to launch for some other reason.

I've added another explicit message here.

Reply:
> I'm guessing you just made the launch command throw to test this?

Yes. I temporarily replaced the condition to always enter that branch